spec/features/monitoring_feature_spec.rb

```diff
@@ -10,6 +10,15 @@
     visit authenticated_root_path
   end
 
+  scenario "It works if there are no minions", js: true do
```

Comment:
This shouldn't have happened. Both [ruby](https://github.com/kubic-project/velum/blob/master/app/views/dashboard/index.html.slim#L5) and [javascript](https://github.com/kubic-project/velum/pull/82/files#diff-398e9dc91ab41e7d183e081c72f8f161L33) only print the message if there are unassigned minions. I thought the problem was when the message was printed once (because unassigned minions exist) and after assigning roles to the minions (not possible through the UI yet), the message wasn't cleared. The latter case is valid although not possible without messing with the console.